pkg/bpf/bpffs.go

```diff
@@ -208,7 +208,10 @@ func mountFS() error {
 
 	}
 	if !isBpffs(mapRoot) {
-		log.Fatalf("BPF: '%s' is not mounted as BPF filesystem.", mapRoot)
+		// TODO currently on minikube this check is failing. Uncomment this log and remove the debug log,
```

Comment:
Can you write this on a different commit?

Reply:
So we can easily do `git revert` once the issue is fixed. The first PR message says it fixes #1472. Therefore my confusion